src/qt/forms/researcherwizardbeaconpage.ui

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>ResearcherWizardBeaconPage</class>
+ <widget class="QWizardPage" name="ResearcherWizardBeaconPage">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>630</width>
+    <height>480</height>
+   </rect>
+  </property>
+  <property name="sizePolicy">
+   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
+    <horstretch>0</horstretch>
+    <verstretch>0</verstretch>
+   </sizepolicy>
+  </property>
+  <property name="windowTitle">
+   <string>Beacon Advertisement</string>
+  </property>
+  <property name="title">
+   <string>Beacon Advertisement</string>
+  </property>
+  <property name="subTitle">
+   <string>A beacon is a special type of Gridcoin transaction that links a BOINC CPID to your wallet. After sending a beacon, the network tracks your BOINC statistics to calculate research rewards.</string>
```

Comment:
This can be simplified. Not sure that we really need to mention its a transaction. Also might want to say account instead of CPID since many people aren't going to know what a CPID is. 
```suggestion
   <string>A beacon links a BOINC account to your wallet. After sending it, the network tracks your BOINC statistics to calculate research rewards.</string>
```

Reply:
You can't say "it" there. That is called a general reference error in English. Does the "it" refer to the beacon or the account? Also, account is a squishy term in BOINC.

Reply:
I know an account in BOINC is kind of a weird thing, but I don't think calling it a CPID will mean much to many of the people new to Gridcoin. What about putting it the CPID part in parenthesis so that people who don't know what it is still have an idea that it's related to your account? Maybe something like this:
```suggestion
   <string>A beacon links a BOINC account (through CPID) to your wallet. After sending a beacon, the network tracks your BOINC statistics to calculate research rewards.</string>
```

Reply:
How about "identity"?

```suggestion
   <string>A beacon is a special type of Gridcoin transaction that links your BOINC identity (CPID) to your wallet. After sending a beacon, the network tracks your BOINC statistics to calculate research rewards.</string>
```

Reply:
I like that. Not quite correct in the purest sense, but the best in one word.

Reply:
I think I would prefer `account (through CPID)`, but I guess I'd be okay with identity. When I think of identity I would generally think of it being used with personal identifiable information.

Reply:
Let's just drop "CPID" from the description. We can see from the body of this page that a beacon is related to the CPID. I'm going with this:

```suggestion
   <string>A beacon links your BOINC accounts to your wallet. After sending a beacon, the network tracks your BOINC statistics to calculate research rewards.</string>
```